src/core/InputDropdown/style.ts

```diff
@@ -306,7 +293,6 @@ interface DetailsAndCounter extends CommonThemeProps {
 export const StyledLabel = styled("span", {
   shouldForwardProp: (prop: string) => !doNotForwardProps.includes(prop),
 })`
-  ${labelFontBodyXs}
```

Comment:
Here's where we're removing the original override for all InputDropdowns' counters to be size xs (now xs is the default which they inherit, so it doesn't need to be set here)